### PR TITLE
Refactor styles to use logical properties for positioning

### PR DESCRIPTION
**Pull request by Gargron**
This simplifies style overrides needed for right-to-left layouts and ensures no components are forgotten in there.

### CODE REVIEW

**Review by ClearlyClaire — APPROVED** [review]
This looks good overall, but there are a few changes that may not be intended (I honestly don't know which version is better) with the RTL layouts:

## Before

image

## After

image

**Comment by llamacucumber** [comment]
This broke galleries in chrome on Veganism Social

Chrome: image

Safari: 
image

Problem diff line:
image

Inline start not supported in Chrome:
image

image

**Comment by ClearlyClaire** [comment]
Ah, good catch. I checked support for other properties but not this one. We may need to revert this specific change.

File: app/javascript/styles/mastodon/admin.scss [hunk 1]
```diff
@@ -778,7 +783,7 @@ a.name-tag,
 
 .speech-bubble {
   margin-bottom: 20px;
-  border-left: 4px solid $ui-highlight-color;
+  border-inset-inline-start: 4px solid $ui-highlight-color;
```
Comment (ClearlyClaire) [inline app/javascript/styles/mastodon/admin.scss:786]: Does this actually exist?

Comment (nschonni) [reply app/javascript/styles/mastodon/admin.scss:786]: https://github.com/mastodon/mastodon/blob/77bd32a26abbf8b49caf35dcdf170e312e48a277/stylelint.config.js#L16 can be used as a check for some of that stuff

Comment (Gargron) [reply app/javascript/styles/mastodon/admin.scss:786]: Oh, I messed up replacing absolute positioning properties (left/right) and failed to notice. My bad!

**Pull request merged**